samples/README.md

```diff
@@ -6,13 +6,7 @@ If you are interested more in the larger list of pipelines samples we are testin
 [KFP Tekton User Guide](/guides/kfp-user-guide) is a guideline for the possible ways to develop and consume Kubeflow Pipeline with Tekton. It's recommended to go over at least one of the methods in the user guide before heading into the KFP Tekton Samples.
 
 ## Prerequisites
-- Install [Kubeflow 1.0.2+](https://www.kubeflow.org/docs/started/getting-started/) and connect the cluster to the current shell with `kubectl`
-- Install [Tekton 0.16.3](https://github.com/tektoncd/pipeline/releases/tag/v0.16.3)
-    - For KFP, we shouldn't be modifying the default work directory for any component. Therefore, please run the below command to disable the [home and work directories overwrite](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#customizing-the-pipelines-controller-behavior) from Tekton default.
-        ```shell
-        kubectl patch cm feature-flags -n tekton-pipelines -p '{"data":{"disable-home-env-overwrite":"true","disable-working-directory-overwrite":"true"}}'
-        ```
-- Install [Kubeflow pipeline with Tekton backend](/guides/kfp_tekton_install.md)
+- Install [Kubeflow 1.3+](https://www.kubeflow.org/docs/started/getting-started/) or install [standalone kfp-tekton 0.7.0+](/guides/kfp_tekton_install.md#standalone-kubeflow-pipelines-with-tekton-backend-deployment). Then connect the cluster to the current shell with `kubectl`
```

Comment:
The first part of this sentence before the "or" makes it sound like users can choose to only install Kubeflow (without Tekton or the KFP-Tekton backend)?

Reply:
I updated the wording to ask users to deploy kfp with tekton backend.